src/App/DataSource/Email/Email.php

```diff
@@ -111,6 +113,17 @@ public function getOptions()
 		);
 	}
 
+	public function getInboundFields()
+	{
+		return [
+			'From' => 'text',
```

Comment:
I'm not 100% clear why these keys are capitalized rather than say matching the key on the message payload.

I wonder if these were more like the `options` definition they might be more extendable ie
```
'somefield' => [
  'type' => 'text',
  'description' => 'Some field',
]
```

Reply:
No idea, they were originally capitalised. Perhaps it was related to earlier incarnations of the message system when it was first built.